github_connector_odoo/models/odoo_module_version.py

```diff
@@ -0,0 +1,504 @@
+
+    # Compute Section
+    @api.depends(
+        "repository_id.organization_id.github_name",
+        "repository_id.name",
+        "repository_branch_id.name",
+        "repository_branch_id.local_path",
+        "full_module_path",
+    )
+    def _compute_github_url(self):
+        for version in self:
+            repo_id = version.repository_id
+            version.github_url = (
+                "https://github.com/{organization_name}/"
+                "{repository_name}/tree/{branch_name}/{rest_path}".format(
+                    organization_name=repo_id.organization_id.github_name,
+                    repository_name=repo_id.name,
+                    branch_name=version.repository_branch_id.name,
+                    rest_path=(version.full_module_path or "")[
+                        len(version.repository_branch_id.local_path or "") + 1 :
+                    ],
+                )
+            )
+
+    @api.depends("repository_branch_id.repository_id.name")
+    def _compute_odoo_type(self):
+        for version in self:
+            repository_name = version.repository_branch_id.repository_id.name
+            if repository_name.startswith("l10n-"):
+                version.odoo_type = "localization"
+            elif repository_name.startswith("connector-"):
+                version.odoo_type = "connector"
+            elif repository_name.startswith("vertical-"):
+                version.odoo_type = "verticalization"
+            else:
+                version.odoo_type = "other"
+
+    @api.depends("technical_name", "repository_branch_id.complete_name")
+    def _compute_complete_name(self):
+        for version in self:
+            version.complete_name = (
+                version.repository_branch_id.complete_name
+                + "/"
+                + version.technical_name
+            )
+
+    @api.depends("description_rst")
+    def _compute_description_rst_html(self):
+        for version in self:
+            if version.description_rst:
+                try:
+                    output = publish_string(
+                        source=version.description_rst,
+                        settings_overrides=self._SETTING_OVERRIDES,
+                        writer=MyWriter(),
+                    )
+                except Exception:
+                    output = (
+                        "<h1 style='color:red;'>"
+                        + _("Incorrect RST Description")
+                        + "</h1>"
+                    )
+            else:
+                output = html_sanitize(
+                    "<h1 style='color:gray;'>" + _("No Version Found") + "</h1>"
+                )
+            version.description_rst_html = html_sanitize(output)
+
+    @api.depends("depends")
+    def _compute_dependency_module_ids(self):
+        module_obj = self.env["odoo.module"]
+        for version in self:
+            modules = []
+            for module_name in version.depends.split(","):
```

Comment:
When testing, if I try to create a version manually, I get this error:

```
AttributeError: 'bool' object has no attribute 'split'
```

Are you meant to do this manually? If not, can a new error message be displayed to avoid hitting the error?